### ISSUE TITLE
Difference between Help information in Checker view and Reports in HTML & Excel

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#872]
Types of Help information are inconsistent between Checker view and Help viewed from Report excel spreadsheet columns and Help page linked from report.

### Included in Checker view, but missing from **_Report columns_**:
- [ ] **_ReasonID_** 
- [ ] **_failue message_** 

### Included in Checker view, but missing from **_Help page_** linked from Report (See images below):
- [ ] **_ReasonID_** 
- [ ] **_failue message_** in title area at top of page
- [ ] **_RuleID_** 
- [ ] **_Issue Type & icon_** 
- [ ] **_Element location_**

### Included in Report colums, but missing from Checker view:
- [ ] **_Toolkit level_**  
- [ ] **_WCAG level_**

Example of difference 
### Help in Checker view
![image](https://user-images.githubusercontent.com/16193609/167202506-c41e11b0-8022-43fe-bcc7-de973947ba91.png)

### Help linked from Reports
![image](https://user-images.githubusercontent.com/16193609/167202675-e9bc9f3b-6356-4264-8a82-4e187c1c17e7.png)

**Comment by shunguoy** [comment]
caused by a bug in the extension report generation.

**Comment by philljenkins** [comment]
This issue is about which types and kinds of information (columns in the spreadsheets) reported, not about the values or counts reported.

**Comment by philljenkins** [comment]
Example from an excel report
[Accessibility_Report-Altoro Accessibility Testing Site.xlsx](https://ibm.ent.box.com/file/1188831682660)

![image](https://user-images.githubusercontent.com/16193609/231496876-0143efb9-a314-4504-bed0-e56087a8a19b.png)

**Comment by philljenkins** [comment]
Example from an HTML report
[Accessibility_Report-Altoro Accessibility Testing Site.html](https://ibm.ent.box.com/file/1188820887023)

![image](https://user-images.githubusercontent.com/16193609/231497808-21e98b64-ddb2-44b1-91e6-d9a2423aef96.png)

**Comment by tombrunet** [comment]
Phill will open a separate design issue regarding the level content and whether or not that should be included.

**Comment by philljenkins** [comment]
Open issue https://github.ibm.com/IBMa/E2E/issues/5685
for designing remaining tasks above
